lib/webhook-proxy.js

```diff
@@ -1,14 +1,38 @@
 module.exports = ({url, port, path, logger}) => {
+  
   try {
-    const SmeeClient = require('smee-client')
-
-    const smee = new SmeeClient({
+    var goodUrl = true; //url is considered to be true;
+    const SmeeClient = require('smee-client');
+    const valid= ValidURL(url);
+    if(!valid)
+    {
+      goodUrl = false; //url is invalid
+      throw "check your Webhook proxy URL";
+    }
+    else
+    {
+      const smee = new SmeeClient({
       source: url,
       target: `http://localhost:${port}${path}`,
       logger
     })
     return smee.start()
+
+    }
+    
   } catch (err) {
-    logger.warn('Run `npm install --save-dev smee-client` to proxy webhooks to localhost.')
+    if(goodUrl) logger.warn('Run `npm install --save-dev smee-client` to proxy webhooks to localhost');
+    else logger.warn(err);
   }
 }
+
+//function to validate the url (taken from stack-overflow)
+function ValidURL(str) {
```

Comment:
can you please change this to `validURL`, lowercase `V`? Functions that start with an uppercase letter are typically reserved for classes 😀

Reply:
On it.